docs/src/user-guide/guides-using-object-storage/archive-storage.md

```
@@ -0,0 +1,64 @@
+      credentials:
+        access_key_id: "<aws-access-key-id>"
+        secret_access_key: "<aws-secret-access-key>"
+
+  # archive_output's other config keys
+```
+
+Replace the fields in angle brackets (`<>`) with the appropriate values:
+
+* `s3_config` configures both the S3 bucket where archives should be stored and the credentials
+  for accessing it.
+  * `<region-code>` is the AWS region [code][aws-region-codes] for the bucket.
```

Markdownlint (0.37.0)

Missing link or image reference definition: "aws-region-codes"
Reference links and images should use a label that is defined

(MD052, reference-links-images)